cloud/endagaweb/models.py

```diff
@@ -103,15 +104,20 @@ def new_user_hook(sender, instance, created, **kwargs):
         """
         if created and instance.username != settings.ANONYMOUS_USER_NAME:
             profile = UserProfile.objects.create(user=instance)
-            network = Network.objects.create()
+
+            # To avoid duplicate names while running setup_test_db
+            # network = Network.objects.create()
```

Comment:
We haven't removed any of the old code which is changed or updated in case it is still required by your team.
Now removed.